inference-engine/src/transformations/src/transformations/convert_precision.cpp

```diff
@@ -412,7 +412,7 @@ bool fuse_type_to_constant(std::shared_ptr<Node> & node, element::Type to, const
         }
 
         new_const->validate_and_infer_types();
-        if (constant->get_output_target_inputs(0).size() == consumers.size()) {
+        if (constant->get_output_size() == consumers.size()) {
```

Comment:
`constant->get_output_size()` returns number of output ports and for Constant it will always return 1. So this check looks weird.
I tried to remember the purpose of `constant->get_output_target_inputs(0).size() == consumers.size()` check and the reason was to avoid non-unique names in Function. But now I think we can remove this check and always set name to new_constant.

Reply:
@GlebKazantaev  At line 114 there is comment:
```
  // As Constant operations can be shared between multiple nGraph Functions so before
  // changing precision we need to understand which Constant consumers belongs
  // to the current nGraph Function
```
so it was probably the reason. In such a case `constant->get_output_size()` can return value greater than 0 or I miss sth?

Reply:
`constant->get_output_size()` return just the number of output ports of operation and for Constant it is always 1.
And yes, Constant can be shared between two functions but it doesn't prevent us to rename its copy. In worst case we will have multiple Constant with the same name but with different precisions. But as we support  non-unique name in Function it is no more a problem.

Reply:
ohh, right, I've mistaken meaning of `get_output_size` in this case ;) so I just removed the condition